spack/package.py

```diff
@@ -27,7 +27,7 @@ class Arbor(CMakePackage, CudaPackage):
 
     # https://arbor.readthedocs.io/en/latest/install/build_install.html?highlight=requirements#compilers
     conflicts('%gcc@:8.3')
-    conflicts('%clang@:8.0')
+    conflicts('%clang@:7.0')
```

Comment:
`7.0` will probably allow clang 7.1+ to be used. Would just `7` block and clang with major version 7 to be blocked?

Reply:
Right, I thought I typed 7. Fixed!